apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/GraphQLCompiler.kt

```diff
@@ -75,6 +75,6 @@ open class GraphQLCompiler {
   companion object {
     const val FILE_EXTENSION = "graphql"
     val OUTPUT_DIRECTORY = listOf("generated", "source", "apollo")
-    const val APOLLOCODEGEN_VERSION = "0.10.8"
+    const val APOLLOCODEGEN_VERSION = "0.10.9"
```

Comment:
Did you check if this changes any of the IR files? (you can run `./gradlew apollo-compiler:updateTestIRFiles`